crowd_anki/representation/note.py

```diff
@@ -122,12 +129,17 @@ def save_to_collection(self, collection, deck, model_map_cache, import_config):
         self.anki_object.__dict__.update(self.anki_object_dict)
         self.anki_object.mid = note_model.anki_dict["id"]
         self.anki_object.mod = anki.utils.intTime()
-        self.anki_object.flush()
 
         if new_note:
-            collection.addNote(self.anki_object)
-        elif not import_config.ignore_deck_movement:
-            self.move_cards_to_deck(deck.anki_dict["id"])
+            if saner_note_addition:
+                collection.add_note(self.anki_object, deck.anki_dict["id"])
```

Comment:
does this mean we can directly specify the deck id, and so don't need the `did` magic above anymore?

Reply:
Yes. 

I had wrapped the `did` magic so it only ran for old Anki versions, for compatibility, and now have completely removed it. (Unless I missed part of it.)